registration/tests/utils/utils.py

```diff
@@ -87,8 +87,7 @@ def create_test_children(
                 role=Student.CHILD,
                 information={
                     "8": fake.random_element(elements=genders),
-                    "9": fake.date_this_decade().strftime("%m/%d/%Y"),
```

Comment:
@Lambo-Liu sorry I missed this earlier - would you mind adding a DOB field with a fake date for the dummy children?